From 54886659d15b41eda8d29c374956c65122015aad Mon Sep 17 00:00:00 2001
From: Clayton Coleman <ccoleman@redhat.com>
Date: Thu, 29 Oct 2020 13:55:57 +0100
Subject: [PATCH] UPSTREAM: <carry>: Always test PDB's during service upgrade
 test

The upstream can't enable this, but we need to do so in order to
properly validate that cluster upgrades retain availability.

OpenShift-Rebase-Source: 0385e16c6d7
---
 test/e2e/upgrades/network/services.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/e2e/upgrades/network/services.go b/test/e2e/upgrades/network/services.go
index 44036c58a57..83f6d407d32 100644
--- a/test/e2e/upgrades/network/services.go
+++ b/test/e2e/upgrades/network/services.go
@@ -41,7 +41,7 @@ type ServiceUpgradeTest struct {
 // Name returns the tracking name of the test.
 func (ServiceUpgradeTest) Name() string { return "service-upgrade" }
 
-func shouldTestPDBs() bool { return framework.ProviderIs("gce", "gke") }
+func shouldTestPDBs() bool { return true }
 
 // Setup creates a service with a load balancer and makes sure it's reachable.
 func (t *ServiceUpgradeTest) Setup(ctx context.Context, f *framework.Framework) {
-- 
2.43.0

